precompiles/bank/setup_test.go

```diff
@@ -39,6 +44,9 @@ type PrecompileTestSuite struct {
 func TestPrecompileTestSuite(t *testing.T) {
 	s = new(PrecompileTestSuite)
 	suite.Run(t, s)
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Bank Extension Suite")
```

Comment:
as @facs95 pointed out it might be cleaner to separate the unit test suite from the integration test suite, you can check on the PR I had opened for the ERC20 integration tests 👍